crcsim/experiment/combine_tests.py

```diff
@@ -0,0 +1,121 @@
+import json
+from pathlib import Path
+from typing import Dict
+
+import fire
+
+
+def combine_tests_in_params(
+    params: Dict, test1: str, test2: str, how: str = "serial"
```

Comment:
Parallel should probably be the default. Seems like a more likely way CRC screens would be combined to me.